packages/flutter/test/widgets/spacer_test.dart

```diff
@@ -9,14 +9,22 @@ void main() {
   testWidgets('Spacer takes up space.', (WidgetTester tester) async {
```

Comment:
It would be useful to keep the existing tests as-is as those use cases should continue to work. Instead, new tests should be added for the new widget.

Reply:
Done.